src/ScheduledTasks/Client/IScheduleHandle.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.ScheduledTasks;
+
+/// <summary>
+/// Represents a handle to a schedule, allowing operations on a specific schedule instance.
+/// </summary>
+public interface IScheduleHandle
```

Comment:
It may be better to use an abstract class than an interface, so that we can add new methods or overloads in the future as a non-breaking change.

Also, with regards to the name: to me the term 'Handle' suggests a local resource, but here, the resource is inside a remote service. In the Azure Storage SDK, such "handles for remote resources" are usually called clients (e.g. `BlobClient`, `BlobContainerClient`). So this could be called `ScheduleClient`, following that logic. I don't have a strong opinion on that though.